examples/fleet/README.md

```diff
@@ -4,6 +4,345 @@ This directory contains a number of examples of how to deploy applications to mu
+
+To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+```
+
+Each fleet member Azure resource has a corresponding MemberCluster Kubernetes custom resource in the hub cluster.
+View the list of MemberClusters (add `-o yaml` if you want to view more details):
+
+```shell
+kubectl get memberclusters
+```
+
+The output is similar to:
+
+```console
+NAME           JOINED   AGE
+aks-member-1   True     11m
+aks-member-2   True     11m
+aks-member-3   True     11m
+```
+
+### 7. Get credentials to access member clusters
+
+```shell
+az aks get-credentials -g ${GROUP} -n ${MEMBER1} --file member1
+KUBECONFIG=member1 kubectl cluster-info
+
+az aks get-credentials -g ${GROUP} -n ${MEMBER2} --file member2
+KUBECONFIG=member2 kubectl cluster-info
+
+az aks get-credentials -g ${GROUP} -n ${MEMBER3} --file member3
+KUBECONFIG=member3 kubectl cluster-info
+```
+
+The output is similar to:
+
+```console
+Merged "aks-member-1" as current context in member1
+Kubernetes control plane is running at https://aks-member-fleet-group-8ecadf-a7af50a9.hcp.westcentralus.azmk8s.io:443
+CoreDNS is running at https://aks-member-fleet-group-8ecadf-a7af50a9.hcp.westcentralus.azmk8s.io:443/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+Metrics-server is running at https://aks-member-fleet-group-8ecadf-a7af50a9.hcp.westcentralus.azmk8s.io:443/api/v1/namespaces/kube-system/services/https:metrics-server:/proxy
+
+To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+
+Merged "aks-member-2" as current context in member2
+Kubernetes control plane is running at https://aks-member-fleet-group-8ecadf-4e806f1a.hcp.westcentralus.azmk8s.io:443
+CoreDNS is running at https://aks-member-fleet-group-8ecadf-4e806f1a.hcp.westcentralus.azmk8s.io:443/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+Metrics-server is running at https://aks-member-fleet-group-8ecadf-4e806f1a.hcp.westcentralus.azmk8s.io:443/api/v1/namespaces/kube-system/services/https:metrics-server:/proxy
+
+To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+
+Merged "aks-member-3" as current context in member3
+Kubernetes control plane is running at https://aks-member-fleet-group-8ecadf-d771b794.hcp.eastus.azmk8s.io:443
+CoreDNS is running at https://aks-member-fleet-group-8ecadf-d771b794.hcp.eastus.azmk8s.io:443/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+Metrics-server is running at https://aks-member-fleet-group-8ecadf-d771b794.hcp.eastus.azmk8s.io:443/api/v1/namespaces/kube-system/services/https:metrics-server:/proxy
+
+To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+```
+
+To easily switch among multiple member clusters, you can use a command like the following (i.e., always add `KUBECONFIG=memberx` as a prefix to `kubectl`):
```

Comment:
nit: I think the recommended/cleaner way for some time is to use `--kubeconfig <path>`, instead of KUBECONFIG env var.